micromixin-transformer/src/main/java/org/stianloader/micromixin/transform/internal/annotation/MixinShadowAnnotation.java

```diff
@@ -52,7 +55,18 @@ public static <T0 extends ClassMemberStub> MixinShadowAnnotation<T0> parse(@NotN
                 throw new MixinParseException("Unimplemented option for @Shadow: " + name);
             }
         }
-        return new MixinShadowAnnotation<T0>(prefix, aliases);
+
+        boolean isMutable = false;
+        if (allAnnotations != null) {
+            for (AnnotationNode currAnn : allAnnotations) {
+                if (currAnn.desc.equals("Lorg/spongepowered/asm/mixin/Mutable;")) {
```

Comment:
Would be great if you were to throw a `MixinParseException` if `!currAnn.values.isEmpty()`